tasks/deploy/config/mainnet-fork/reth-30day.ts

```diff
@@ -0,0 +1,71 @@
+import { formatEther, parseEther } from "viem";
+import {
+    HyperdriveInstanceConfig,
+    MAINNET_RETH_ADDRESS,
+    getLinkerDetails,
+    normalizeFee,
+    parseDuration,
+    toBytes32,
+} from "../../lib";
+
+const CONTRIBUTION = parseEther("500");
+
+export const MAINNET_FORK_RETH_30DAY: HyperdriveInstanceConfig<"RETH"> = {
+    name: "RETH_30_DAY",
+    prefix: "RETH",
+    coordinatorAddress: async (hre) =>
+        hre.hyperdriveDeploy.deployments.byName("RETH_COORDINATOR").address,
+    deploymentId: toBytes32("RETH_30_DAY"),
+    salt: toBytes32("0x666"),
+    extraData: "0x",
+    contribution: CONTRIBUTION,
+    fixedAPR: parseEther("0.05"),
+    timestretchAPR: parseEther("0.05"),
+    options: {
+        destination: process.env.ADMIN! as `0x${string}`,
+        asBase: false,
+        extraData: "0x",
+    },
+    prepare: async (hre) => {
+        await hre.run("fork:mint-reth", {
+            address: (await hre.getNamedAccounts())["deployer"],
+            amount: formatEther(CONTRIBUTION),
+        });
+        let vaultSharesToken = await hre.viem.getContractAt(
+            "MockRocketPool",
+            MAINNET_RETH_ADDRESS,
+        );
+        let pc = await hre.viem.getPublicClient();
+        // TODO: Investigate why significant additional allowance is needed for deposit (10 ether is too little)
+        let tx = await vaultSharesToken.write.approve([
+            hre.hyperdriveDeploy.deployments.byName("RETH_COORDINATOR").address,
+            CONTRIBUTION + parseEther("100"),
+        ]);
```

Comment:
This is more confusing to me since RETH isn't a rebasing token. Is this just legacy from the steth scripts?